src/VirtoCommerce.Platform.Web/wwwroot/js/app/security/blades/common/scope-value-pick-from-simple-list.js

```diff
@@ -3,21 +3,27 @@ angular.module('platformWebApp')
     var blade = $scope.blade;
 
     function initializeBlade() {
-        blade.dataPromise.then(function (data) {
-            blade.isLoading = false;
-
-            _.each(blade.currentEntity.assignedScopes, function (x) {
-                var store = _.find(data, function (y) { return x.scope === y.id; });
-                if (store) {
-                    store.$selected = true;
-                }
-            });
-
-            blade.currentEntities = data;
-        }, function (error) {
-            bladeNavigationService.setError('Error ' + error.status, blade);
+        blade.selectedIds = _.map(blade.currentEntity.assignedScopes, x => x.scope);
+        $scope.items = _.map(blade.currentEntity.assignedScopes, function (x) {
+            var result = {};
+            result.id = x.scope;
+            result.name = x.label;
+            return result;
         });
```

Comment:
how about restyling:

 $scope.items = _.map(blade.currentEntity.assignedScopes, x => {
            id: x.scope,
            name: x.label
        });

Reply:
This does not work. 'Name' is a const.